src/include/daos_srv/pool.h

```diff
@@ -34,6 +35,7 @@ struct ds_pool {
 	struct pool_map	       *sp_map;
 	uint32_t		sp_map_version;	/* temporary */
 	uint64_t		sp_reclaim;
+	tier_policy_t		sp_policy; 	/* pool tiering policy */
```

Comment:
Is an integer sufficient for describing a policy? Maybe we need a data structure to contain more parameters like what io size for SCM, what io size for performance tier, what io size for capacity tier, etc. (and the io size criteria could be different for client IO and aggregation IO)

If we expand it to a data structure, we may need to duplicate it in each ds_pool_child. (in this shared nothing architecture, 'ds_pool' is maintained & accessed by xstream 0, 'ds_pool_child' is for each VOS xstream.